colour/contrast/__init__.py

```diff
@@ -0,0 +1,141 @@
+        Retinal illuminance :math:`E` in Trolands.
+    N_max : numeric or array_like, optional
+        {:func:`colour.contrast.contrast_sensitivity_function_Barten1999`},
+        Maximum number of cycles :math:`N_{max}` over which the eye can
+        integrate the information.
+    T : numeric or array_like, optional
+        {:func:`colour.contrast.contrast_sensitivity_function_Barten1999`},
+        Integration time :math:`T` in seconds of the eye.
+    X_0 : numeric or array_like, optional
+        {:func:`colour.contrast.contrast_sensitivity_function_Barten1999`},
+        Angular size :math:`X_0` in degrees of the object in the x direction.
+    Y_0 : numeric or array_like, optional
+        {:func:`colour.contrast.contrast_sensitivity_function_Barten1999`},
+        Angular size :math:`Y_0` in degrees of the object in the y direction.
+    X_max : numeric or array_like, optional
+        {:func:`colour.contrast.contrast_sensitivity_function_Barten1999`},
+        Maximum angular size :math:`X_{max}` in degrees of the integration
+        area in the x direction.
+    Y_max : numeric or array_like, optional
+        {:func:`colour.contrast.contrast_sensitivity_function_Barten1999`},
+        Maximum angular size :math:`Y_{max}` in degrees of the integration
+        area in the y direction.
+    k : numeric or array_like, optional
+        {:func:`colour.contrast.contrast_sensitivity_function_Barten1999`},
+        Signal-to-noise (SNR) ratio :math:`k`.
+    n : numeric or array_like, optional
+        {:func:`colour.contrast.contrast_sensitivity_function_Barten1999`},
+        Quantum efficiency of the eye :math:`n`.
+    p : numeric or array_like, optional
+        {:func:`colour.contrast.contrast_sensitivity_function_Barten1999`},
+        Photon conversion factor :math:`p` in
+        :math:`photons\\div seconds\\div degrees^2\\div Trolands` that
+        depends on the light source.
+    phi_0 : numeric or array_like, optional
+        {:func:`colour.contrast.contrast_sensitivity_function_Barten1999`},
+        Spectral density :math:`\\phi_0` in :math:`seconds degrees^2` of the
+        neural noise.
+    sigma : numeric or array_like, optional
+        {:func:`colour.contrast.contrast_sensitivity_function_Barten1999`},
+        Standard deviation :math:`\\sigma` of the line-spread function
+        resulting from the convolution of the different elements of the
+        convolution process.
+    u : numeric
+        {:func:`colour.contrast.contrast_sensitivity_function_Barten1999`},
+        Spatial frequency :math:`u`, the cycles per degree.
+    u_0 : numeric or array_like, optional
+        {:func:`colour.contrast.contrast_sensitivity_function_Barten1999`},
+        Spatial frequency :math:`u_0` in :math:`cycles\\div degrees` above
+        which the lateral inhibition ceases.
+
+    Returns
+    -------
+    ndarray
+        Contrast sensitivity function :math:`S`.
```

Comment:
I did not read the paper, so I'm wondering why a "function" has the type ndarray?
Might be good to elaborate a bit more what the array represents. The documentaiton of the function doesn't seem to explain it much more.

Reply:
Ah, nicely spotted, should read _Contrast sensitivity :math:`S`._